components/director/Dockerfile

```diff
@@ -22,11 +24,13 @@ WORKDIR /app
 # Copy binary
 #
 
+RUN apk --no-cache add curl
```

Comment:
You may also need the certificates to be present in your container as the fetcher communicates with the external API that should be done using https.